configure

```diff
@@ -2485,35 +2485,49 @@ sub write_cmake_file {
     # These flags only appear in the %opts map when specified by the user.
     my %enabled_by_default = map {$_, 1} qw(
       built-in-topics
-      content-subscription
       content-filtered-topic
+      content-subscription
+      debug
+      inline
       multi-topic
-      query-condition
-      ownership-profile
-      ownership-kind-exclusive
       object-model-profile
+      ownership-kind-exclusive
+      ownership-profile
       persistence-profile
-      inline
+      query-condition
     );
 
     my %disabled_by_default = map {$_, 0} qw(
+      ipv6
       security
+      static
     );
 
     my %ignore = map {$_, 0} qw(
+      ace
+      ace-github-latest
```

Comment:
I'm not that familiar with how this is used... but could we just assume that a key is an `ignore` key if it's not in one of the other hashes?